10_2_es_bipedal.py

```diff
@@ -0,0 +1,82 @@
+# Evolution Strategies BipedalWalker-v2
+# https://blog.openai.com/evolution-strategies/
+# converges at around iter 100 in 5 minutes
+# for testing model set reload=True
+
+import gym
+import numpy as np
+import cPickle as pickle
```

Comment:
Are you using Python2?

cPickle is renamed to pickle in python3
no need to import cPickle